config/sidekiq_scheduler.yml

```diff
@@ -2,64 +2,22 @@
 # over all workers. The `every` option will run the job once per worker instance
 # at the specified interval.
 
-AccountLoginStatisticsJob:
-  cron: "0 6 * * * America/New_York"
-  description: "Gather account login statistics for statsd"
 
-AppealsApi::DailyErrorReport:
-  cron: "0 23 * * MON-FRI America/New_York"
-  class: AppealsApi::DailyErrorReport
-  description: "Daily report of appeals errors"
+###########################
+# PERIODIC JOBS ARE REPLACING SIDEKIQ SCHEDULER
 
-AppealsApi::DailyStuckRecordsReport:
-  cron: '0 8 * * MON-FRI America/New_York'
-  class: AppealsApi::DailyStuckRecordsReport
-  description: "Daily report of all stuck appeals submissions"
+# Please add new periodic jobs in lib/periodic_jobs.rb
 
-AppealsApi::DecisionReviewReportDaily:
-  cron: "0 23 * * MON-FRI America/New_York"
-  class: AppealsApi::DecisionReviewReportDaily
-  description: "Daily report of appeals submissions"
-
-AppealsApi::DecisionReviewReportWeekly:
-  cron: "0 23 * * SUN America/New_York"
-  class: AppealsApi::DecisionReviewReportWeekly
-  description: "Weekly report of appeals submissions"
 
 AppealsApi::HigherLevelReviewCleanUpWeekOldPii:
   every: ['24h', {first_in: '45m'}]
   class: AppealsApi::HigherLevelReviewCleanUpWeekOldPii
   description: "Remove PII of HigherLevelReviews that have 1) reached one of the 'completed' statuses and 2) are a week old"
```

Comment:
Is there a reason you left a few of these jobs in this file instead of moving them to the new `periodic_jobs.rb` file?

Reply:
The teams will have to add these jobs to the periodic jobs. This job has specs that will fail if moved.